src/plugins/cosmos/components/modals/Staking/views/UnstakeBroadcast.tsx

```diff
@@ -0,0 +1,157 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/layout'
+import { Button, Link, ModalCloseButton, Text as CText, Tooltip } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { Asset } from '@shapeshiftoss/types'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { SlideTransition } from 'components/SlideTransition'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type UnstakeBroadcastProps = {
+  assetId: CAIP19
+  cryptoUnstakeAmount: BigNumber
+  fiatRate: BigNumber
+  isLoading: boolean
+}
+
+export enum Field {
+  FeeType = 'feeType'
+}
+
+export type UnstakeBroadcastParams = {
+  [Field.FeeType]: chainAdapters.FeeDataKey
+}
+
+const DEFAULT_VALIDATOR_NAME = 'Shapeshift Validator'
+
+// TODO: Wire up the whole component with staked data
+export const UnstakeBroadcast = ({
+  assetId,
+  cryptoUnstakeAmount,
+  fiatRate,
+  isLoading
+}: UnstakeBroadcastProps) => {
+  const { cosmosStaking } = useModal()
+
+  const translate = useTranslate()
+
+  const handleClose = cosmosStaking.close
```

Comment:
Nitpick, we could just pass `cosmosStaking.close` directly but this is a pointer and has no performances difference compared to using the original one so I'm agnostic to this.

Reply:
done !